### PR TITLE
Middle section

### PR DESCRIPTION
**Pull request by wuyepabdul**
- Added projects that i have worked on

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -83,6 +82,125 @@ <h2>Software Developer</h2>
           <img src="./icons/work.png" alt="image" />
         </div>
       </section>
+
+      <!-- My Recent work -->
+      <section class="works-section align-center">
+        <h2 class="align-left justify-center">My Recent Work</h2>
+        <hr />
```
Comment (Emmyn5600) [inline index.html]: - [x] This should be h1 tag not h2

File: index.html [hunk 2]
```diff
@@ -83,6 +82,125 @@ <h2>Software Developer</h2>
...
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+
+              <div class="tags d-flex space-between">
+                <div class="align-center">css</div>
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
```
Comment (Emmyn5600) [inline index.html]: - [x] This should be wrapped by `ul` tag.
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 3]
```diff
@@ -83,6 +82,125 @@ <h2>Software Developer</h2>
...
```
Comment (Emmyn5600) [inline index.html]: - [x] This should be wrapped by `ul` tag.
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 4]
```diff
@@ -83,6 +82,125 @@ <h2>Software Developer</h2>
...
+              </div>
```
Comment (Emmyn5600) [inline index.html]: - [x] This should be wrapped by `ul` tag.
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 5]
```diff
@@ -83,6 +82,125 @@ <h2>Software Developer</h2>
...
```
Comment (Emmyn5600) [inline index.html]: - [x] This should be wrapped by `ul` tag.
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @wuyepabdul 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
Changes required :writing_hand:  
* #### You are missing the `About myself` section , and it should be done in this milestone, kindly work on it.
 * #### Please check the comments under the review too
     
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: APPROVED ✔✔**
Hello @wuyepabdul 👋🏽
Your project is complete!  There is nothing else to say. It's time to merge it.
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**